### ISSUE TITLE
Comments & Guidance column collapse is not in tab order

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#374]
Comments & Guidance column collapse control is not accessible via keyboard. It is not part of the tab order.

Applicable WCAG Standard(s)
[2.1.1 Keyboard](http://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable) (Level A)
